src/Radiation/rrtmgp_interface.jl

```diff
@@ -0,0 +1,120 @@
+using RRTMGP
+using RRTMGP: RRTMGPGridParams
+using RRTMGP.AngularDiscretizations
+using RRTMGP.Fluxes
+using RRTMGP.Optics
+using RRTMGP.GrayUtils
+using RRTMGP.Sources
+using RRTMGP.BCs
+using ClimaParams
+
+import RRTMGP.Parameters.RRTMGPParameters
+import RRTMGP.AtmosphericStates
+import RRTMGP.RTE: NoScatLWRTE, TwoStreamLWRTE, NoScatSWRTE, TwoStreamSWRTE
+
+function RRTMGP.RRTMGPGridParams(grid::RectilinearGrid; isothermal_boundary_layer::Bool = false)
+    nlay = grid.Nz
+    ncols = grid.Nx * grid.Ny
+    context = context_from_arch(grid.architecture)
+    return RRTMGPGridParams(context, nlay, ncols, isothermal_boundary_layer)
+end
+
+function RRTMGP.AtmosphericStates.GrayAtmosphericState(
+    grid::RectilinearGrid;
+    temperature::Field,
+    pressure::Field,
+    lat::Field,
```

Comment:
Can you explain how we have both a `grid` argument as well as a `lat` argument (eg latitude?) In the case of any curvilinear grid, we can extract the latitudes from the grid itself so would not need an independent input. But I might be misunderstanding how this should work. I could also imagine that radiation works only with column grids (eg `RectilinearGrid` with `(Flat, Flat, Bounded)` topology)